graphql/src/handler.rs

```diff
@@ -0,0 +1,23 @@
+/// This will be used in both tests and 
+/// various GraphQL resolver for thehost level(chain, mempool, net...)
+/// which will be implemented in this crate as well.
+pub fn handle_gql_query<T: async_graphql::ObjectType + Send + Sync + 'static>(
+    runtime: &tokio::runtime::Handle,
+    root: T,
+    query: &str,
+    variables: &str,
+) -> String {
+    let variables = if let Ok(s) = (|| -> Result<_, ()> {
+        Ok(async_graphql::Variables::parse_from_json(async_graphql::serde_json::from_str(variables).map_err(|_| ())?)
```

Comment:
How about?
```suggestion
        let json_variables = async_graphql::serde_json::from_str(variables).map_err(|_| ())?;
        let variables = async_graphql::Variables::parse_from_json(json_variables).map_err(|_| ())?;
        Ok(variables)
```

I feel hard to read code if a line contains multiple error handlings.